react-sdk/src/providers/tambo-thread-provider.tsx

```diff
@@ -143,11 +175,11 @@ export const TamboThreadContext = createContext<TamboThreadContextProps>({
    *
    */
   sendThreadMessage: () => {
-    throw new Error("advance not implemented");
+    throw new Error("sendThreadMessage not implemented");
   },
-  generationStage: GenerationStage.IDLE,
-  generationStatusMessage: "",
-  isIdle: true,
+  /**
```

Comment:
I don't see how leaving this out of this provider is going to change the re-rendering?

thanks to the final useMemo() in GenerationStageProvider, these values won't change